misc/container-health/Dockerfile

```diff
@@ -10,7 +10,7 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
-FROM busybox:1.32.0
+FROM public.ecr.aws/docker/library/busybox:1.32.0
```

Comment:
Yeah we could. I just didn't want to make unnecessary changes since `AmazonLinuxLegacy2018` branch should only be receiving the very necessary updates.

Reply:
discussed offline, no need in order to prevent making unnecessary changes